app/views/dashboard/associated_users/destroy.js.coffee

```diff
@@ -32,7 +32,7 @@ $("#documents-table").bootstrapTable()
 $(".service-requests-table").bootstrapTable()
 
 $('.service-requests-table').on 'all.bs.table', ->
-	$(this).find('.selectpicker').selectpicker() #Find descendant selectpickers
+       $(this).find('.selectpicker').selectpicker() #Find descendant selectpickers
```

Comment:
@jayhardee9 
I have absolutely no clue. I couldn't get rid of this edit. I tried changing it to multiple space lengths and couldn't get it to be removed. I have a refactor coming right after this PR so I will make that update then.